openfisca_core/scripts/test_perf_fancy_indexing.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
```

Comment:
Rename file to `measure_performances_fancy_indexing` to unify with other filenames in this directory?